### DISCUSSION TITLE
Time for a new release?

### DISCUSSION THREAD

**Discussion by callumrollo** [c-proof/pyglider#147] [Ideas]
The last release was in June 2022. We've made a lot of improvements since then, including dbdreader, using polars for seaexplorer and better nan filtering. Is it time for a version 0.05?

This would also be a good time to update the documentation, particularly ` get_example_data` which currently returns a 404 from the link to https://cproof.uvic.ca/pyglider-example-data/pyglider-example-data.zip

**Comment by jklymak** [comment]
So what are we doing for get_example_data?  Is it just data distributed with the package now?

**Comment by callumrollo** [comment]
I think the method with `get_example_data` is good. It means we don't need to add lots of data to the package. It's just that the zip file is not currently available on the cproof website. Could you re-upload it? Alternatively, we could host the zip file somewhere else, like on github or zenodo

**Comment by brianverwey** [comment]
Adding to this thread as a new user trying out different slocum processing software, it would be super nice if the get_example_data returned an example dataset to play with.

**Comment by jklymak** [comment]
@brianverwey Sorry, I didn't know the link was down.  

https://cproof.uvic.ca/pyglider-example-data/pyglider-example-data.zip should work now.

**Comment by brianverwey** [comment]
Thanks Jody, I appreciate the quick response!
